src/styles/shared/_buttons.scss

```diff
@@ -235,13 +236,16 @@
     }
 
     &:focus {
-      box-shadow: 0 0 0 border-width('thick') var(--p-border-secondary);
-      @include no-focus-ring;
+      box-shadow: 0 0 0 border-width('base') var(--p-border-secondary);
+      @include focus-ring($style: 'focused');
     }
 
     &:active {
       box-shadow: 0 0 0 border-width('base') var(--p-border-secondary);
       background: var(--p-surface-pressed);
+      &::after {
+        box-shadow: none;
```

Comment:
Just a thought... we end up overwriting things about the ring in a few places. It doesn't feel right because it's so far detached for the source. If we change the ring, this will just add tech debt because chances are it will be left behind. Adding more arguments to the focus-ring with defaults would allow us to keep all the focus ring CSS in 1 location.

Reply:
Good point! I've been chatting with Alex about refactoring button css once the design language has been rolled out. I think that would be a good time to address things like this as we'll have a better idea of how the focus ring needs to be used